string robot_ip
bool status
---
bool is_ready